runtime/runtime_impl_jetty9/src/main/java/com/google/apphosting/runtime/jetty9/JettyRequestAPIData.java

```diff
@@ -286,7 +287,7 @@ public JettyRequestAPIData(
       traceContext = TraceContextProto.getDefaultInstance();
     }
 
-    String finalUserIp = userIp;
+    String finalUserIp = NORMALIZE_INET_ADDR ? HostPort.normalizeHost(userIp) : userIp;
```

Comment:
[nit] Just went and checked the source of this change and came across this CL: https://critique.corp.google.com/cl/400610752. Curious if this flag is ever used by the customers? There is a typo in the naming of the property in that it is is called `nomalize_inet_addr` instead of `normalize_inet_addr`